app/graphql/types/world_index_type.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+module Types
+  class WorldIndexType < Types::EditionType
+    class WorldLocation < Types::BaseObject
```

Comment:
Will it be easy to move this type out of here and into it's own type file, if we decide to migrate world location pages to use GraphQL?

Reply:
This is actually a different kind of world location, just used for the world index page, so we're putting it in here to avoid a naming clash if we do create WorldLocation edition types.

Reply:
Cool, that makes sense.  Thanks for explaining.